Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -648,15 +648,18 @@ public double GetNamedSize(NamedSize size, Type targetElementType, bool useOldSi
 
 			public async Task<Stream> GetStreamAsync(Uri uri, CancellationToken cancellationToken)
 			{
-				using (var client = new HttpClient())
-				using (HttpResponseMessage response = await client.GetAsync(uri, cancellationToken))
+				using (var client = new HttpClient())				
```

Comment:
Have you considering using a single static instance of HttpClient across Forms (both on iOS and Android)? In general, it's more efficient to maintain a single active instance than create and dispose multiple instances.

There is some info here that relates to the issue you're fixing: https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/

If we maintain a single instance, then maybe all of this work is unnecessary.

Reply:
I tried making HTTPClient static and the same thing happens.

I agree looking into a static instance of HttpClient would be useful but I don't think it's related to this issue.

That blog is talking about issues that come up from making lots of requests and it causing socket exhaustion right? In this case it's a single request that's failing

Also in the comments of that one Oren links to this
http://byterot.blogspot.com/2016/07/singleton-httpclient-dns.html

So it seems like it's kind of a rabbit hole :-)